Content.Shared/Ghost/GhostComponent.cs

```diff
@@ -0,0 +1,117 @@
+using Content.Shared.Actions;
+using Content.Shared.Actions.ActionTypes;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Ghost
+{
+    [RegisterComponent, NetworkedComponent]
+    [AutoGenerateComponentState]
+    public sealed partial class GhostComponent : Component
```

Comment:
try adding access (if something blows up then remove it since fixing that would be a little out of scope)